pkg/kv/kvserver/raft_transport.go

```diff
@@ -347,9 +341,13 @@ func (t *RaftTransport) RaftMessageBatch(stream MultiRaft_RaftMessageBatchServer
 	errCh := make(chan error, 1)
 
 	// Node stopping error is caught below in the select.
-	if err := t.stopper.RunAsyncTask(
-		stream.Context(), "storage.RaftTransport: processing batch",
-		func(ctx context.Context) {
+	taskCtx, cancel := t.stopper.WithCancelOnQuiesce(stream.Context())
+	defer cancel()
+	if err := t.stopper.RunAsyncTaskEx(
+		taskCtx,
+		stop.TaskOpts{TaskName: "storage.RaftTransport: processing batch",
```

Comment:
Surprised crlfmt/gofmt doesn't complain -- would feel more readable if moved to the next line like you've done below.
```
stop.TaskOpts{
  TaskName: ...
```